### PR TITLE
Add missing git-wordiness script

### PR DESCRIPTION
**Pull request by nilium**
# Description

Add the missing git-wordiness script.

# Type of changes

- [x] A helper script
- [ ] A link to an external resource like a blog post or video
- [ ] Text cleanups/updates

# Checklist:

- [ ] All new and existing tests pass.
- [x] Any scripts added use `#!/usr/bin/env interpreter` instead of potentially platform-specific direct paths (`#!/bin/sh` is an ok exception)
- [x] Scripts are marked executable
- [x] Scripts _do not_ have a language file extension unless they are meant to be sourced and not run standalone. No one should have to know if a script was written in bash, python, ruby or whatever. Not including file extensions makes it easier to rewrite the script in another language later without having to change every reference to the previous version.
- [ ] I have added a credit line to README.md for the script
- [ ] If there was no author credit in a script added in this PR, I have added one.
- [x] I have confirmed that the link(s) in my PR are valid.
- [x] I have read the **CONTRIBUTING** document.

# License Acceptance

- [x] This repository is Apache version 2.0 licensed (some scripts may have alternate licensing inline in their code) and by making this PR, I am contributing my changes to the repository under the terms of the Apache 2 license.

### CODE REVIEW

File: bin/git-wordiness [hunk 1]
```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+#
+# Source: https://gist.github.com/nilium/5083d036576559fb7bb9dafc822adb29
+
+# FIXME: Names with non-ASCII characters can throw off alignment of
```
[the remark this answers is not in the document]
Comment (nilium) [reply bin/git-wordiness:5]: I'm not fixing this right now.

File: bin/git-wordiness [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+export shanames
+export shaprefix
+
+export logquote="$(if [[ "${#logargs[@]}" -gt 0 ]] ; then printf ' %q' "${logargs[@]}" ; fi)"
+
+export fmt
+if [[ $head == 1 ]]
+then
+        printf "$hfmt" COMMITS WPC LINES WORDS BYTES NAME
```
[the remark this answers is not in the document]
Comment (nilium) [reply bin/git-wordiness:127]: No, this is intentional, the variable is a format string.

File: bin/git-wordiness [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                logargs=("$arg" "$@")
+                break
+                ;;
+        esac
+done
+
+export shanames
+export shaprefix
+
+export logquote="$(if [[ "${#logargs[@]}" -gt 0 ]] ; then printf ' %q' "${logargs[@]}" ; fi)"
```
[the remark this answers is not in the document]
Comment (nilium) [reply bin/git-wordiness:122]: I have no idea what it's asking for here.

[the remark this answers is not in the document]
Comment (unixorn) [reply bin/git-wordiness:119]: It's a bit overzealous sometimes.

**Review by unixorn — COMMENTED** [review]
lgtm

**Pull request merged**